conf.py

```diff
@@ -18,6 +20,38 @@
 ### Extensions
 ############################################################
 
+#############################################################
+# Display the contributor
+
+def get_contributors_for_file(github_url, github_folder, pagename, page_source_suffix):
+    edit_url = f"{github_url}/edit/{pagename}{page_source_suffix}"
+    parsed_url = urlparse(edit_url)
```

Comment:
Why do you need `edit_url`? Can you just parse `github_url` directly, or will that cause any problems?